### DISCUSSION TITLE
Some Words are Cut Off From The Page

### DISCUSSION THREAD

**Discussion by indigofficial** [community/community#21365] [Other Feature Feedback, Questions, & Ideas]
<p>Hi there!</p>
<p>I’m building a <a href="https://infinite-dreams.github.io/demo" rel="noopener nofollow ugc">demo holiday deal website</a>, and the <a href="https://github.com/infinite-dreams/infinite-dreams.github.io/blob/master/demo.html" rel="noopener nofollow ugc">files</a> are in GitHub as well. If you see the first link (which leads to the demo website) on your phone, scroll down and you might notice that the B’s on the left side is cut off.</p>
<p>I spent hours trying to fix this problem and I can’t solve this. Could someone please help me with this problem?</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>They look fine to me - _if_ my browser window is wide enough. They are cut off when I start to shrink the width of my browser window.</p>
<p>Using the Developer Tools in Chrome, I looked at the web site. Here is what it shows for your “row” div:</p>
<p><img src="https://user-images.githubusercontent.com/9809727/181063891-9823076d-d10f-4b37-9a0e-e3864d251e9d.png" alt="Capture.PNG" data-base62-sha1="8IpEzpVhiS5nSJuJrYzPb8iWByl" width="339" height="452"></p>
<p>As you hopefully can see from the ‘Styles’ section, “margin-left” is set to “-15px”. Not sure how that is set, but a zero or positive margin value will probably make the left-hand entries fully visible in a narrow browser window.</p>

**Comment by indigofficial** [comment]
<p><span class="mention">@fire_eggs</span> I did that, and it moved a little bit, and then I put in centered-items:</p>
<pre><code>centered-items {
                align-content: center;
                margin-top: 7rem;
                margin-left: 20rem;
            }</code></pre>
